libs/ui-watt/src/lib/foundations/typography/+storybook/storybook-typography-overview.component.ts

```diff
@@ -25,110 +25,147 @@ interface Typography {
     letterSpacing: '0',
-    lineHeight: '36px*',
   },
   {
     name: 'Headline 5',
     html: typographyHtmlSnippets.h5.tag,
-    size: '20px',
-    weight: 'Semibold (600)',
-    letterCase: 'All caps',
+    size: 'M',
+    weight: 'Bold',
+    letterCase: 'Sentence',
     letterSpacing: '0',
-    lineHeight: '30px*',
   },
   {
-    name: 'Lead (text-l)',
+    name: 'large',
     html: typographyHtmlSnippets.textL.class,
-    size: '18px',
-    weight: 'Semibold (600)',
+    size: 'L',
+    weight: 'Regular',
     letterCase: 'Sentence',
     letterSpacing: '0',
-    lineHeight: '27px*',
   },
   {
-    name: 'Body (text-m)',
+    name: 'normal',
     html: typographyHtmlSnippets.bodyTextM.tag,
-    size: '16px',
-    // eslint-disable-next-line sonarjs/no-duplicate-string
-    weight: 'Regular (400)',
+    size: 'M',
+    weight: 'Regular',
     letterCase: 'Sentence',
     letterSpacing: '0',
-    lineHeight: '24px*',
   },
   {
-    name: 'Small (text-s)',
+    name: 'small',
     html: typographyHtmlSnippets.textS.tag,
-    size: '14px',
-    weight: 'Regular (400)',
+    size: 'S',
+    weight: 'Regular',
+    letterCase: 'Sentence',
+    letterSpacing: '0',
+  },
+  {
+    name: 'large-highlighted',
+    html: typographyHtmlSnippets.textLHighlighted.class,
+    size: 'L',
+    weight: 'Bold',
+    letterCase: 'Sentence',
+    letterSpacing: '0',
+  },
+  {
+    name: 'normal-highlighted',
+    html: typographyHtmlSnippets.bodyTextMHighlighted.tag,
+    size: 'M',
+    weight: 'Bold',
     letterCase: 'Sentence',
     letterSpacing: '0',
-    lineHeight: '21px*',
   },
   {
-    name: 'Extra small (text-xs)',
-    html: typographyHtmlSnippets.textXs.class,
-    size: '12px',
-    weight: 'Regular (400)',
+    name: 'small-highlighted',
+    html: typographyHtmlSnippets.textSHighlighted.tag,
+    size: 'S',
+    weight: 'Bold',
     letterCase: 'Sentence',
     letterSpacing: '0',
-    lineHeight: '18px*',
   },
   {
     name: 'Button',
     html: typographyHtmlSnippets.button.storybook,
-    size: '14px',
-    weight: 'Semibold (600)',
-    letterCase: 'All caps',
-    letterSpacing: '1.25px',
-    lineHeight: '16px',
+    size: 'M',
+    weight: 'Bold',
+    letterCase: 'Sentence',
+    letterSpacing: '0',
   },
   {
     name: 'Label',
     html: typographyHtmlSnippets.label.class,
-    size: '12px',
-    weight: 'Semibold (600)',
+    size: 'S',
+    weight: 'Bold',
     letterCase: 'All caps',
-    letterSpacing: '1.5px',
-    lineHeight: '14.4px',
+    letterSpacing: '1.25px',
+  },
+  {
+    name: 'Normal Link',
+    html: typographyHtmlSnippets.link.class,
+    size: 'M',
+    weight: 'Regular',
+    letterCase: 'Sentence',
+    letterSpacing: '0',
+  },
+  {
+    name: 'Small Link',
+    html: typographyHtmlSnippets.linkS.class,
+    size: 'S',
+    weight: 'Regular',
+    letterCase: 'Sentence',
+    letterSpacing: '0',
+  },
+];
+
+const typeScaleSmall: Typography[] = [
+  {
+    name: 'headline 1',
```

Comment:
nitpick: Normalize text by starting with capital letter - `Headline 1`. Check for other similar texts in the file.